documentation/source/users/rmg/installation/ubuntuSubsystem.rst

```diff
@@ -0,0 +1,43 @@
+.. _ubuntuSubsystem:
+
+*****************************************************
+Installing RMG in the Ubuntu Subsystem on Windows 10
+*****************************************************
+
+Installing the Ubuntu Subsystem
+===================================
+
+1. Follow the instructions provided by Microsoft to install the Ubuntu subsystem, available `here
+<https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_. The basic steps include enabling the Windows Linux
+subsystem from a powershell **run as an administrator**, and downloading the latest LTS version of Ubuntu
+*from the Windows store*. We highly recommend Ubuntu over the other Linux distros, as Ubuntu is used by many of the
+RMG developers.
```

Comment:
instead of `We highly recommend Ubuntu over the other Linux distros, as Ubuntu is used by many of the RMG developers.`, maybe just: `We recommend Ubuntu (for which these instructions were made) over the other Linux distributions`?

Reply:
Done!